### PR TITLE
✨(ui): create Company-Explore directory and layout

### PR DESCRIPTION
**Pull request by Thivesshj**
created the company-explore library. and designed the desktop-layout with angular, angular-materials, and angular flex-layout to match the wire frame.

### CODE REVIEW

**Comment by arneschreuder** [comment]
Please resolve conflicts.

File: apps/client/src/app/app.module.ts [hunk 1]
```diff
@@ -19,7 +20,7 @@ import { AppComponent } from './app.component';
       // or after 30 seconds (whichever comes first).
       registrationStrategy: 'registerWhenStable:30000',
     }),
-    ClientShellFeatureModule,
+    ClientShellFeatureModule,ClientCompanyExploreFeatureModule,
```
Comment (arneschreuder) [inline apps/client/src/app/app.module.ts:23]: Please move this import to be imported by the Shell. The ShellFeatureModule should be the only thing imported by app.module.ts

File: libs/client/company-explore/feature/src/lib/company-explore-feature/company-explore-content/company-explore-card/company-explore-card.component.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<mat-card class="card" fxLayout="column">
```
Comment (arneschreuder) [inline libs/client/company-explore/feature/src/lib/company-explore-feature/company-explore-content/company-explore-card/company-explore-card.component.html:1]: Just a comment. Nx has feature libs and UI libs. You have it correct. A feature component belongs in the feature lib as you have it. Try see if there is a need to make UI libs. UI Libs are reusable "dumb" components. These include like searchbars, dropdowns etc. Just a comment. No changes needed.

File: libs/client/company-explore/feature/src/lib/materials/company-explore-material.module.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import { NgModule } from '@angular/core';
```
Comment (arneschreuder) [inline libs/client/company-explore/feature/src/lib/materials/company-explore-material.module.ts:1]: Perhaps make a shared module and export/import all of these there. Like a MaterialShared Module or something. Then other people can also use it.

**Review by arneschreuder — CHANGES_REQUESTED** [review]
This is exciting. Nice. A few small changes. Also. You need a routing module. Get this merged in and then add routing. Specifically: https://angular.io/guide/lazy-loading-ngmodules

File: apps/client/src/app/app.module.ts (hunk 1 above)
Comment (Thivesshj) [reply apps/client/src/app/app.module.ts]: removed import from the app.module.ts

File: libs/client/company-explore/feature/src/lib/materials/company-explore-material.module.ts (hunk 3 above)
Comment (Thivesshj) [reply libs/client/company-explore/feature/src/lib/materials/company-explore-material.module.ts:1]: speaking to the person in charge of the shared libs, and will add it after my merge

File: libs/client/company-explore/feature/src/lib/company-explore-feature/company-explore-content/company-explore-card/company-explore-card.component.html (hunk 2 above)
Comment (Thivesshj) [reply libs/client/company-explore/feature/src/lib/company-explore-feature/company-explore-content/company-explore-card/company-explore-card.component.html:1]: I will be sure to look into that. thank you.

**Comment by arneschreuder** [comment]
I'm copy pasting this on all the open PRs. Ignore if already seen or applied: https://github.com/COS-301/graduates/discussions/3?sort=new#discussioncomment-2540319

**Comment by arneschreuder** [comment]
This PR still needs to update with `upstream/develop`

**Comment by Thivesshj** [comment]
I updated my PR and my branch is now up to date with the upstream COS-301:develop .Thank you :sparkles::thumbsup::sparkles:

**Review by arneschreuder — APPROVED** [review]

**Pull request merged**